client/src/services/geocode.js

```diff
@@ -0,0 +1,21 @@
+async function getCoords(place) {
+  let apikey = process.env.REACT_APP_OPEN_CAGE_DATA_KEY;
+  let api_url = "https://api.opencagedata.com/geocode/v1/json";
+  let request_url =
+    api_url +
+    "?" +
+    "q=" +
+    encodeURIComponent(place) +
+    "&key=" +
+    apikey +
+    "&language=es" +
+    "&pretty=1" +
+    "&countrycode=pe" +
+    "&no_annotations=1";
```

Comment:
Could you please explain what this `piece of code` does?

Reply:
That is the way to get a request from [Open Cage Data API](https://opencagedata.com/api)